release/src/main/scripts/build_release_candidate.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This script will create a Release Candidate, includes:
+# 1. Build and stage java artifacts
+# 2. Stage source release on dist.apache.org
+# 3. Stage python binaries
+# 4. Create a PR to update beam-site
+
+set -e
+
+LOCAL_CLONE_DIR=build_release_candidate
+LOCAL_JAVA_STAGING_DIR=java_staging_dir
+LOCAL_PYTHON_STAGING_DIR=python_staging_dir
+LOCAL_WEBSITE_UPDATE_DIR=website_update_dir
+LOCAL_PYTHON_DOC=python_doc
+LOCAL_JAVA_DOC=java_doc
+LOCAL_WEBSITE_REPO=beam_website_repo
+
+USER_REMOTE_URL=
+USER_GITHUB_ID=
+GIT_REPO_URL=https://github.com/apache/beam.git
+ROOT_SVN_URL=https://dist.apache.org/repos/dist/dev/beam
+GIT_BEAM_ARCHIVE=https://github.com/apache/beam/archive
+GIT_BEAM_WEBSITE=https://github.com/apache/beam-site.git
+
+PYTHON_ARTIFACTS_DIR=python
+BEAM_ROOT_DIR=beam
+WEBSITE_ROOT_DIR=beam-site
+
+
+echo "================Setting Up Environment Variables==========="
+echo "Which release version are you working on: "
+read RELEASE
+RELEASE_BRANCH=release-${RELEASE}
+echo "Which release candidate number(1-3) are you going to create: "
```

Comment:
nit; number is not limited to 1-3

Reply:
Changed in https://github.com/apache/beam/pull/6059/commits/622873b71ad59255fa8212547cf935a2cc7b7c0d